Correo - Base64

FJaque opened the pull request.

FJaque committed:
Agregado envío de correo

FJaque committed:
Conversiones base64

psnull reviewed (changes requested):
Todo bien, solo falta actualizar desde dev

FJaque committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into correo-base64

FJaque committed:
Corregida conversión base64

Se elimina el método que obtenía los valores desde la DB
Se mantiene el método que recibe los valores por parámetro

psnull committed:
Agregado path relativo y archivo de prueba

psnull committed:
Separadas funcionalidades email y base64 en otras clases
